proto/observer/observer.proto

```diff
@@ -15,6 +15,13 @@ enum ObservationType {
   TSSKeySign = 4;
 }
 
+enum ObserverUpdateReason {
+  option (gogoproto.goproto_enum_stringer) = true;
+  EmptyObserverUpdateReason = 0;
```

Comment:
```suggestion
  Undefined = 0;
```

Reply:
Thought on this? I think `EmptyObserverUpdateReason` is unnecessarily long since the field name is already embedded in `ObserverUpdateReason`